### PR TITLE
[WIP] Remove babosa workaround since it has been fixed upstream

### PR DESCRIPTION
**Pull request by milch**
Since https://github.com/norman/babosa/pull/42 has been merged in Babosa, as soon as a new version of the gem is released this PR can be merged. 

This PR removes the workaround that we needed because of the unicode-display_width gem that was introduced in #6991 (see also https://github.com/janlelis/unicode-display_width/issues/4).

@mfurtak Could you look over this to make sure it takes care of everything, since you introduced the workaround in #7243?

### CODE REVIEW

**Comment by hjanuschka** [comment]
awesome, got the notification today that babosa PR was merged, nice to see this hack being removed.! 🚀

File: spaceship/lib/spaceship/client.rb [hunk 1]
```diff
@@ -3,7 +3,6 @@
 require 'faraday_middleware'
 require 'fastlane/version'
 require 'logger'
-require 'spaceship/babosa_fix'
```
Comment (mfurtak) [inline spaceship/lib/spaceship/client.rb]: Do we need to replace this with a `require 'babosa'` if we're taking out the fix class?

**Comment by parndt** [comment]
Open source is fun.. I use fastlane for app deployments and needed to use `unicode-display_width` to determine the correct length of non-English strings, which is where I saw the bug filed against babosa which I help to maintain to merging the PR made a lot of sense. Enjoy, and thanks for fastlane! 👍

File: spaceship/lib/spaceship/client.rb (hunk 1 above)
Comment (milch) [inline spaceship/lib/spaceship/client.rb]: Right, good point. Thanks!

**Comment by KrauseFx** [comment]
Hey @milch, that's an awesome PR, thanks 👍 What's the best way to test it? To create Chinese apps, or apps with emojis using _produce_? I heard @hjanuschka has some Chinese apps on his account now.

**Comment by hjanuschka** [comment]
@KrauseFx i have like 100+ chinese app's 😨 - but it totally would give it a try and create another one. but since the last babosa release is from 2015 i am not really sure if this will ever be released?

once done, please ping me - and i ll create another chinese app 🎉

**Comment by KrauseFx** [comment]
Alright, let's leave this WIP and blocked for now, if the status doesn't change within the next few weeks, we should probably just close the PR for now

**Comment by ohayon** [comment]
Hey everyone, looks like babosa is not pushing any updates these days. I am going to close this PR for now and we can update down the road if necessary. Thanks everyone!

**Pull request closed without merging**